appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -985,12 +985,16 @@
   @Description("Error shown when a new component type would be the same as a component instance name")
   String sameAsComponentInstanceNameError();
 
-  @DefaultMessage("Component name cannot be any of the following: CsvUtil, Double, Float, " +
+  @DefaultMessage("Name cannot be any of the following: CsvUtil, Double, Float, " +
       "Integer, JavaCollection, JavaIterator, KawaEnvironment, Long, Short, SimpleForm, String, " +
-      "Pattern, YailList, YailNumberToString, YailRuntimeError")
+      "Pattern, YailList, YailNumberToString, YailRuntimeError, abstract, continue, for, new, " +
+          "switch, assert, default, goto, package, synchronized, boolean, do, if, private, this, break, " +
```

Comment:
The spacing switches from 6 to 10 here. It should be 6 to be consistent with the rest of the sources.

Reply:
Whitespace fixed, and tests added.